drivers/SmartThings/zigbee-fan/src/itm-fan-light/init.lua

```diff
@@ -0,0 +1,201 @@
+-- Copyright 2023 philh30
```

Comment:
```suggestion
-- Copyright 2024 SmartThings
```

Reply:
The suggested content han been revised.
Also, we are setting up and checking the test folder commented by greens.
We are going to add test code to src folder.